### ISSUE TITLE
file upload widget

### ISSUE THREAD

**Issue by bennol** [MetaModels/contao-frontend-editing#36]
Trying to use the `file` attribute for frontend-editing results in the following error:
```
 ContaoCommunityAlliance\\DcGeneral\\Exception\\DcGeneralRuntimeException(code: 0): Widget was not build for property [PROPERTY NAME]
``` 
As listed in https://github.com/MetaModels/contao-frontend-editing/issues/15 this feature does not yet seem to be implemented. Are there any working forks for Contao 4.4 or examples of how to implement this?

**Comment by zonky2** [comment]
@baumannsven work at a new version - pls call him...

**Comment by bennol** [comment]
@baumannsven do you have any news on this? Or can you point me into the right direction?

**Comment by zonky2** [comment]
implemented in MM 2.2
